kontrakter/dom/begjaeringDom/arbeidsversjon/begjaeringDom.schema.json

```diff
@@ -0,0 +1,1297 @@
+          "properties": {
+            "status": {
+              "type": "string",
+              "enum": ["VARSLET_DOK", "IKKE_ONSKET_FORSVARER"]
+            },
+            "beskrivelse": {
+              "type": "string"
+            }
+          },
+          "required": ["status"],
+          "additionalProperties": false
+        },
+        "varsleBistandsadvokat": {
+          "type": "string"
+        },
+        "begrenseOffentlighet": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/begrenseOffentlighetKrav"
+          }
+        },
+        "ansvarligPaatalejurist": {
+          "$ref": "#/definitions/paataleJurist"
+        },
+        "aktor": {
+          "$ref": "#/definitions/paataleJurist"
+        }
+      },
+      "required": [
+        "varetektsType",
+        "paagrepetTidspunkt",
+        "begrenseOffentlighet",
+        "ansvarligPaatalejurist"
+      ],
+      "additionalProperties": false
+    },
+    "begrenseOffentlighetKrav": {
+      "type": "string",
+      "description": "Liste over ulike begrense offentlighet krav.",
+      "enum": ["LUKKEDE_DOERER", "FORBUD_GJENGIVELSE", "HEMMELIGHOLD"]
+    },
+    "fengsling": {
+      "type": "object",
+      "description": "Hvor lenge i varetekt",
+      "properties": {
+        "varighet": {
+          "$ref": "#/definitions/varighet"
+        }
+      },
+      "required": ["varighet"],
+      "additionalProperties": false
+    },
+    "varighet": {
+      "type": "object",
+      "description": "Varighet på fengsling/restriksjon, antall dager eller frem til hovedforhandling (dato satt i saksinformasjon)",
+      "properties": {
+        "antallDager": {
+          "type": "integer"
+        },
+        "tilHovedforhandling": {
+          "type": "boolean",
+          "const": true
+        }
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "antallDager": {
+              "type": "integer"
+            }
+          },
+          "required": ["antallDager"],
+          "additionalProperties": false
+        },
+        {
+          "properties": {
+            "tilHovedforhandling": {
+              "type": "boolean",
+              "const": true
+            }
+          },
+          "required": ["tilHovedforhandling"],
+          "additionalProperties": false
+        }
+      ]
+    },
+    "forlengelseInfo": {
+      "type": "object",
+      "description": "Hvis det er en forlengelse så referer vi til forrige fengsling, trenger domstolene mer informasjon ?",
+      "properties": {
+        "avgjoerelseId": {
+          "type": "string",
+          "description": "Domstolene sin nøkkel til den forrige fengslingskjennelsen"
+        },
+        "beskrivelse": {
+          "type": "string",
+          "description": "Kanskje referanse til dokument etc"
+        }
+      },
+      "required": ["beskrivelse"],
+      "additionalProperties": false
+    },
+    "siktetInfo": {
```

Comment:
Skal siktetInfo være med i begjæringen? Kommer ikke alle siktede/tiltale i siktelsen/tiltalen?

Reply:
Jo, den er en reserveløsning hvis vi ikke har en strukturert siktelse og for at dere skal kunne vise siktedes informasjon med en gang (hvis det er forsinkelser på siktelsen).